src/Nav/Nav.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { NavLink, Link } from 'react-router-dom';
 import logo from '../assets/clean-beauty-logo.png';
 import './Nav.css';
 
 const Nav = () => {
   return (
     <nav className="nav">
-      <img className="nav-logo" src={logo} alt="clean beauty logo" />
+      <Link to="/">
+        <img className="nav-logo" src={logo} alt="clean beauty logo" />
+      </Link>
       <ul className="nav-links">
         <li>
```

Comment:
A weird thing is happening because you have styled the `li` to look like a button, but only the text itself is clickable, so if you click on the edge of the "button," it doesn't work. I might consider adding the styling directly to the `NavLink` so that the clickable part looks like a button. Let me know if that doesn't make sense!